etcdserver/raft.go

```diff
@@ -195,6 +195,14 @@ func (r *raftNode) resumeSending() {
 	p.Resume()
 }
 
+func enableFastBootstrap(n raft.Node, electionTicks int) {
```

Comment:
How about advanceTicksForElection? Also we should move the comments above the func.